docs/administration-guide/secure-database-connections-with-ssl-certificates.md

```diff
@@ -0,0 +1,63 @@
+## Securing database connections using an SSL certificate
+
+If you'd like to connect your Metabase Cloud or self-hosted instance to a database, you can secure the connection using Secure Socket Layer (SSL) encryption with a certificate.
+
+Why you'd want to do this:
+
+- You're using Metabase Cloud and want to ensure the identity of the data warehouse you're connecting to (e.g., PostgreSQL, MySQL).
+- You're self-hosting Metabase and want to ensure the identity of a data warehouse hosted by an external provider. You can also use this method to ensure you're using the strictest connection parameters when connecting to your application database.
+
+If you're using Metabase Cloud, the application database is handled for you, so you'd only need to secure connections to data warehouses that you add to your Metabase.
+
+### Prerequisites
+
+A database that allows a JDBC connection, as you'll need to use a connection string to specify the certificate you want to use.
+
+### Step 1: Download the root certificate from your provider
+
+If you're running Metabase via a Docker container, you should already have the certificates for AWS and Azure. Otherwise, go to your external provider and find a link to download the root certificate for connecting to your database.
```

Comment:
We can include here that the Azure certificate is under "/app/certs/DigiCertGlobalRootG2.crt.pem" and the AWS RDS one is in "/app/certs/rds-combined-ca-bundle.pem" in Metabase's docker image. If users need any other, they can build their own Docker image